detections/process_ja3_ja3s_map.yml

```diff
@@ -0,0 +1,79 @@
+asset_type: Network
+confidence: low
+creation_date: '2019-10-25'
+data_metadata:
+  data_source:
+    - Carbon Black, Bro
+  data_sourcetypes:
+    - cb:response, bro:ssl
+  providing_technologies:
+    - Carbon Black Response, Bro
```

Comment:
New lines instead commas  FOR data_source data_sourcetypes providing_technologies
Example:
```providing_technologies:
    - Carbon Black Response
    - Bro
```